modules/dataset_access/Makefile

```diff
@@ -0,0 +1,53 @@
+# Copyright 2019 Google LLC
```

Comment:
I don't think we should have a module-specific Makefile. Instead, could we fold any special commands into the standard module Makefile?

Possibly with custom helpers: https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/master/test/task_helper_functions.sh